tabular/src/autogluon/tabular/models/xgboost/xgboost_model.py

```diff
@@ -76,7 +76,9 @@ def _fit(self, X_train, y_train, X_val=None, y_val=None, time_limit=None, num_gp
         else:
             verbose = True
             verbose_eval = 1
-        
+
+        weights = kwargs.get('weights', None)
+        weights_val = kwargs.get('weights_val', None)
```

Comment:
log about weights_val not being used?

Reply:
will add TODO instead, it can be used when scoring this model, so saying it's unused is inappropriate.